tests/test_to_scope.py

```
@@ -0,0 +1,343 @@
+            "version": 1,
+            "deps": [{"name": "peer", "path": str(peer_db)}],
+        }),
+        encoding="utf-8",
+    )
+    monkeypatch.setenv("AELFRICE_KNOWLEDGE_DEPS", str(deps_file))
+
+    # Bootstrap local DB (open/close creates schema).
+    local_store = MemoryStore(str(isolated_db))
+    local_store.close()
+
+    code, err_out = _run("promote", "foreign-01", "--to-scope", "project")
```

Ruff (0.15.12)

[warning] Unpacked variable `err_out` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)